site/source/docs/getting_started/downloads.rst

```diff
@@ -50,7 +50,12 @@ GitHub and set them as :term:`active <Active Tool/SDK>`:
     ./emsdk activate latest
 
     # Activate PATH and other environment variables in the current terminal
-    source ./emsdk_env.sh
+    source /path/to/emsdk/emsdk_env.sh
```

Comment:
The form in the command above looks better to me - why is the longer command preferable?

The shorter form looks better because it is less to type, so there is less room for users to have a typo.